angular/core/components/group_page/group_previous_proposals_card/group_previous_proposals_card.coffee

```diff
@@ -16,3 +16,21 @@ angular.module('loomioApp').directive 'groupPreviousProposalsCard', ->
 
     $scope.canShowMore = ->
       $scope.group.closedMotionsCount > 3
+
+angular.module('loomioApp').filter 'cut', ->
```

Comment:
We have a filter for this called 'truncate', check out `core/filters/truncate_filter.coffee` :)

We use it in subgroups_card.haml